crates/ethportal-api/src/test_utils/types.rs

```diff
@@ -0,0 +1,37 @@
+use serde::{Deserialize, Serialize};
+
+use crate::{
+    types::execution::header_with_proof::HeaderWithProof, ContentValue, ContentValueError,
+    HistoryContentKey, HistoryContentValue, OverlayContentKey, RawContentValue,
+};
+
+/// A common type used in test files.
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct ContentItem<K: OverlayContentKey> {
+    pub content_key: K,
+    #[serde(rename = "content_value")]
+    pub raw_content_value: RawContentValue,
+}
+
+impl<K: OverlayContentKey> ContentItem<K> {
+    pub fn content_value<V: ContentValue<TContentKey = K>>(&self) -> Result<V, ContentValueError> {
+        V::decode(&self.content_key, &self.raw_content_value)
+    }
+}
```

Comment:
Isn't this type already defined in `crates/utils/src/testing.rs`, shouldn't we only define it once?